tests/bruno/e2e/watchtower/force-close-with-pending-tlcs/21-check-balance-node1.bru

```diff
@@ -0,0 +1,40 @@
+meta {
+  name: check balance, the difference should be the tlc2 (6000000) + fee (it may vary, max 5000)
+  type: http
+  seq: 21
+}
+
+post {
+  url: {{CKB_RPC_URL}}
+  body: json
+  auth: none
+}
+
+headers {
+  Content-Type: application/json
+  Accept: application/json
+}
+
+body:json {
+  {
+    "id": 42,
+    "jsonrpc": "2.0",
+    "method": "get_cells_capacity",
+    "params": [
+      {
+        "script_type": "lock"
+      }
+    ]
+  }
+}
+
+script:pre-request {
+  let script = bru.getVar("NODE1_FUNDING_SCRIPT");
+  let body = req.getBody();
+  body.params[0].script = script;
+  req.setBody(body);
+}
+
+assert {
+  Number(BigInt(bru.getVar("NODE1_BALANCE")) - BigInt(res.body.result.capacity)): lt 6005000
```

Comment:
why we can not use `eq` for assertion here?

Reply:
Here the fee will be different depending on the settlement tx size, for example, after running several e2e tests in CI server, it may take 1~2 input cells to fill up the fee, so the whole transaction size will be different, usually the fee is 2000~3000 shannons, so here is a little bit of buffer, to assert the fee is less than 5000.